superset/assets/cypress/selectors/index.js

```diff
@@ -0,0 +1 @@
+export const selectResultsTab = () => cy.get('.SouthPane .ReactVirtualized__Table');
```

Comment:
Why is this one in a separate file?

Reply:
I thought it could possibly be useful to add a concept of re-usable `selectors` that are used in multiple tests and select specific elements. The alternatives would be 

- add a cypress command to select that element (seemed too heavy for this one in particular)
- write the same selector multiple times (this could be fine, if you need to change them they'd be changed in multiple places)

(`"selector"` was borrowed the word from  a common pattern in redux apps to select specific data from the state tree). totally happy to remove this if you + @michellethomas aren't into it.

Reply:
I agree with not writing selector string multiple times. More thinking about where to put the function. My main concern is avoiding a setup that leads to one exploding file that contains selectors from every part of Superset. From the name of the file now it invites people to add selector from all other pages here. Other options would be 
- Rename this to `sqlLab.js` to scope it a bit.
- Put the function in test file itself.

I am more comfortable with either option above.